src/components/VirtualThought.tsx

```diff
@@ -160,13 +157,9 @@ const VirtualThought = ({
     })
   }, [crossContextualKey, onResize, path, thought.id])
 
-  // Read the element's height from the DOM on cursor change and re-render with new height
-  // shimHiddenThought will re-render as needed.
-  useSelectorEffect(updateSize, selectCursor, shallowEqual)
-
```

Comment:
We're running multiline calculations whenever the cursor changes (which should be the only height-affecting effect here?) and trigger `updateSize` in the resulting `useLayoutEffect`, so I think adding the calculation here too adds redundancy and potential race conditions when conflicting with `useLayoutEffect`.

Reply:
`updateMultiline` is triggered when the cursor changes, but how does that lead to `updateSize` being called?

Reply:
It results in `updateSize` being called here by a change in multiline: https://github.com/cybersemics/em/blob/d188a1309f8f630587c772e60b94a032492a0180/src/components/StaticThought.tsx#L112-L114

But you are right, this wouldn't catch all the cases, e.g. a thought wrapping across 3 lines instead of 2 after a cursor change. Adding it back. 👍